src/Features/CSharp/Portable/Completion/CompletionProviders/Snippets/CSharpSnippetCompletionProvider.cs

```diff
@@ -18,7 +19,8 @@ internal class CSharpSnippetCompletionProvider : AbstractSnippetCompletionProvid
     {
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-        public CSharpSnippetCompletionProvider()
+        public CSharpSnippetCompletionProvider(IRoslynLSPSnippetExpander roslynLSPSnippetExpander)
+            : base(roslynLSPSnippetExpander)
         {
         }
     }
```

Comment:
note: it doesn't need ot be part of this PR.  but we shoudl generally have the VB side done at the same time.  that actually will validate that your use of inheritance and hardcoded behavior is correct for both languages.

Reply:
can we do that in the next PR?

Reply:
sounds good.